tests/bsim/bluetooth/host/scan/start_stop/test_scripts/start_stop.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+test_exe="bs_${BOARD_TS}_tests_bsim_bluetooth_host_scan_start_stop_prj_conf"
+simulation_id="start_stop"
+verbosity_level=2
+
+cd ${BSIM_OUT_PATH}/bin
+
+Execute "./${test_exe}" \
+  -v=${verbosity_level} -s="${simulation_id}" -d=0 -testid=scanner \
+  -flash="${simulation_id}.log.bin"
+
+Execute "./${test_exe}" \
+  -v=${verbosity_level} -s="${simulation_id}" -d=1 -testid=periodic_adv \
+  -flash="${simulation_id}.log.bin"
```

Comment:
What's the purpose of the `-flash` argument here?

Reply:
took this from some other test. probably not needed.